src/index.html

```diff
@@ -11,7 +11,153 @@
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="field">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12"
+          required
+          >
+        </label>
+        <label class="field">
+          Full date of birth:
+          <input
+          type="date"
+          name="dateofbirth"
+          required
+          >
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="term"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label>
+          Password:
+          <input
+          type="password"
+          name="password"
+          minlength="5"
+          maxlength="32"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="field">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="lovecats"
+            value="yes"
+            required
+            >Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="lovecats"
+            value="no"
+            required
+            >No
+          </label>
+        </label>
+        <label class="field">
+          What is your favorite color?
+          <input
+          type="color"
+          name="favoriteColor"
+          required
+          >
+        </label>
+        <label class="field">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="bedTime"
+          >
+        </label>
+        <label class="field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label>
+          How do you rate our work?
+          <input
+          type="range"
+          name="ourwork"
+          min="1"
+          max="10"
+          step="1"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label class="field">
+          Comments:
+          <textarea
+          name="comments"
+          autocomplete="off">
+          </textarea>
+        </label>
```

Comment:
```suggestion
          <textarea
            name="comments"
            autocomplete="off"
          ></textarea>
        </label>
```

></textarea> 
для тохо, щоб коректне було введення в формі в `textarea`, потрібно прибрати перенос та всі пробіли між відкриваючим та закриваючим тегом в html
https://user-images.githubusercontent.com/102413088/189622039-79d5a191-2e94-4c5e-bc28-29cd13c9d40e.mov

Reply:
![image](https://user-images.githubusercontent.com/113108366/189649480-43d9ce70-f834-438e-9fee-b6355fb53d1b.png)
Привіт. Мені у такому випадку вибиває помилку при проходженні npm test.

Reply:
Тут я зсунула >< ліворуч на один пробіл, бо у мене не проходив тест npm.